src/lib/hash/skein/skein_512.h

```diff
@@ -27,6 +29,8 @@ class Skein_512 final : public HashFunction {
       */
       Skein_512(size_t output_bits = 512, std::string_view personalization = "");
 
+      Skein_512(const Skein_512& other);
```

Comment:
This is declared but does not seem to be defined anywhere.

Reply:
Good catch! I was running into issues when state-cloning this hash along the way. This is a remnant of a debugging attempt.